packages/flutter/test/material/dialog_test.dart

```diff
@@ -3002,15 +3093,26 @@ class _RestorableDialogTestWidget extends StatelessWidget {
 }
 
 class DialogObserver extends NavigatorObserver {
+  final List<ModalRoute<dynamic>> dialogRoutes = <ModalRoute<dynamic>>[];
   int dialogCount = 0;
```

Comment:
This change would help simplify the logic below.

```suggestion
  int get dialogCount => dialogRoutes.length;
```

(same applies to the Cupertino test as well)

Reply:
i changed, thank you.